plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeConnectorSmokeTest.java

```diff
@@ -1004,15 +1011,11 @@ public void testConvertJsonStatisticsToParquetOnRowType()
                 "{\"numRecords\":1,\"minValues\":{},\"maxValues\":{},\"nullCount\":{}}");
     }
 
-    private List<AddFileEntry> getAddFileEntries(String tableName)
-            throws IOException
-    {
-        return TestingDeltaLakeUtils.getAddFileEntries(getLocationForTable(bucketName, tableName));
```

Comment:
did TestingDeltaLakeUtils.getAddFileEntries become unused method?

Reply:
Yes, it no longer exists